src/chrome-api/index.js

```diff
@@ -18,6 +18,15 @@ export function create() {
 var ChromeManager = {
 
     /**
+     * Install plugin
+     * @param {Object} plugin
+     * @param {Function} plugin.install
+     */
+    install: function (plugin) {
+        plugin.install(this);
```

Comment:
Also, do we really need to explicitly install plugins? Maybe they are always enabled, and if someone does not want it, he just stub methods manually..